.circleci/config.yml

```
@@ -417,7 +417,7 @@ jobs:
           color: "#2EB67D"
           title: $(cat .version)
           message: |
-            Version $(cat .version) of the Codacy chart has been released!
+            Version $(cat .version) of the Codacy chart has been released! You can find it at https://github.com/codacy/chart/releases/tag/$(cat .version)
```

Maybe this could be even simpler, with just the link at the end of the sentence (since it's self-explanatory):

```suggestion
            Version $(cat .version) of the Codacy chart has been released! https://github.com/codacy/chart/releases/tag/$(cat .version)
```